homeassistant/components/ezviz/config_flow.py

```diff
@@ -109,6 +111,10 @@ async def async_step_user(self, user_input=None):
         errors = {}
 
         if user_input is not None:
+
+            if user_input[CONF_REGION] == CONF_CUSTOMIZE:
+                return await self.async_step_user_custom_region_url(user_input)
```

Comment:
Don't pass the input. Store the credentials and ask for the url in the next step. Then validate that before moving on from the next step.

Reply:
Done. Storing in self.context["data"]